pkg/controller/directvolumemigration/rsync.go

```diff
@@ -603,15 +604,25 @@ func (t *Task) createRsyncClientPods() error {
 				},
 			})
 			containers = append(containers, corev1.Container{
-				Name:  fmt.Sprintf("rsync-client-%s", vol),
+				Name:  "rsync-client",
```

Comment:
Thanks this was left over from when we were doing all the clients in multiple containers one pod